src/api/validators/user/user.validator.ts

```diff
@@ -24,6 +24,21 @@ export class UserValidator {
         return request.params.id;
     };
 
+    static logoutToken = async (request: express.Request): Promise<string> => {
```

Comment:
Why this token is sent in the request body?
When the user is already logged in, the token is part of authorization header.

Reply:
This is not jwt token, it is actually firebase token stored in `user_device_details`. We need to destroy this token when user logs out from that device.